components/learn/modules/ROOT/pages/public-staging.adoc

```diff
@@ -1,39 +1,181 @@
 
-=== connection
-infura (explain, get account)
+[[testnet-list]]
+== Available Testnets
 
-=== funds
-get funds (faucet, depends on network)
-hdwallet provider
+There are four test networks available for you to choose, each with their own characteristics:
 
-=== configure w/dotenv
-use dotenv and hdwallet provider
-add network entry to networks.js
+[horizontal]
+Ropsten:: The only proof-of-work testnet. It has unpredictable block times and frequent chain reorganizations. At the same time, it is the chain that most closely resembles mainnet. (id=3)
+Rinkeby:: A proof-of-authority network. This means that new blocks are added by a set of pre-defined trusted nodes, instead of by whichever miner provides a proof-of-work. It only works with Geth clients, and has 15-second block times. (id=4)
+Kovan:: Another proof-of-authority network, but this one runs only with Parity clients, and has 4-second block times. (id=42)
+Goerli:: Also a proof-of-authority network, but compatible with both Geth and Parity clients, with 15-second block times. (id=6)
 
-== send CLI commands
+NOTE: Each network is identified by a numeric ID. Development networks usually have a large random value, while id=1 is reserved for the main Ethereum network.
 
-same as local, but:
-  not instant
-  persistent forever in <network>.json files
-  `from` is important
+It is up to you whether you want to test in a more unpredictable environment, to assess how robust your application is, or a more reliable one, to simplify your testing experience.
 
-note that call is still free and almost instant
+[[connecting-project-to-network]]
+== Connecting a Project to a Testnet
 
-== programmatic CLI commands
+To connect our OpenZeppelin project to a public testnet, we will need to:
 
-same caveats apply, using loader still works
+  . Get hold of a testnet node
+  . Create a new account
+  . Update our networks configuration file
+  . Fund our testing account
 
-next steps:
-  mainnet
+=== Accessing a Testnet Node
+
+While you can spin up your own https://github.com/ethereum/go-ethereum/wiki/Command-Line-Options[Geth] or https://wiki.parity.io/Chain-specification[Parity] node connected to a testnet, the easiest way to access a testnet is via a public node service such as https://infura.io[Infura]. Infura provides access to public nodes for all testnets and the main network, via both free and paid plans.
+
+NOTE: We say a node is _public_ when it can be accessed by the general public, and manages no accounts. This means that it can reply to queries and relay signed transactions, but cannot sign transactions on it own.
+
+Head over to Infura, or your public node provider of your choice, create an account, and jot down your assigned project ID - we will use it later to connect to the network.
```

Comment:
```suggestion
Head over to Infura or a public node provider of your choice, sign up, and jot down your assigned project ID - we will use it later to connect to the network.
```

I'd avoid using 'account' here, given the next section.

Reply:
Good catch